runtime/vm/ContinuationHelpers.hpp

```diff
@@ -74,6 +74,9 @@ class VM_ContinuationHelpers {
 		J9JITGPRSpillArea tempGPRs = continuation->jitGPRs;
 		continuation->jitGPRs = *(J9JITGPRSpillArea*)vmThread->entryLocalStorage->jitGlobalStorageBase;
 		*(J9JITGPRSpillArea*)vmThread->entryLocalStorage->jitGlobalStorageBase = tempGPRs;
+		J9I2JState tempI2J = continuation->i2jState;
+		continuation->i2jState = vmThread->entryLocalStorage->i2jState;
+		vmThread->entryLocalStorage->i2jState = tempI2J;
 		SWAP_MEMBER(oldEntryLocalStorage, J9VMEntryLocalStorage*, vmThread->entryLocalStorage, continuation);
```

Comment:
`vmThread->entryLocalStorage` is being accessed multiple times. It can be stored in a local to reduce the number of pointer dereferences.